static/css/style.css

```diff
@@ -72,47 +72,83 @@ body {
 .fa-heart:hover {
     transform: scale(1.5);
     background-image: linear-gradient(90deg,
-        var(--red), var(--red),
-        var(--orange), var(--orange),
-        var(--yellow), var(--yellow),
-        var(--green), var(--green),
-        var(--blue), var(--blue),
-        var(--purple), var(--purple)
-    );
+            var(--red), var(--red),
+            var(--orange), var(--orange),
+            var(--yellow), var(--yellow),
+            var(--green), var(--green),
+            var(--blue), var(--blue),
+            var(--purple), var(--purple));
     -webkit-background-clip: text;
     -webkit-text-fill-color: transparent;
     cursor: pointer;
     transition: all .6s cubic-bezier(0.86, 0, 0.07, 1);
 }
 
-#chevron-checkbox:checked ~ .chevron-checkbox-label>.fa-heart {
+#chevron-checkbox:checked~.chevron-checkbox-label>.fa-heart {
```

Comment:
Can this be returned to the version that it is above? With the spaces around the `~`.